internal/core/items.rs

```diff
@@ -1248,8 +1249,12 @@ impl Item for ContextMenu {
     ) -> InputEventResult {
         match event {
             MouseEvent::Pressed { position, button: PointerEventButton::Right, .. } => {
-                self.show.call(&(crate::api::LogicalPosition::from_euclid(position),));
-                InputEventResult::EventAccepted
+                if self.enabled() {
```

Comment:
you should also prevent the ContextMenu to have effect on android, and on the menu key.

You can do 
```rust
 if !self.enabled() {
      return InputEventResult::EventIgnored
 }
```

at the beginning of this function.

And also in key_event